scripts/west_commands/runners/jlink.py

```diff
@@ -5,9 +5,10 @@
 '''Runner for debugging with J-Link.'''
 
 import os
-import tempfile
-import sys
+import shlex
 import shutil
+import sys
+import tempfile
```

Comment:
Thanks for keeping these in alphabetical order. You're the best.

Reply:
:+1: Ain't nothing but a `:sort`!